### PR TITLE
fix(client): duplicates and zero length units

### LINKED ISSUE #45885: CSS - Maintainability issues in multiple files
[CodeFactor](https://www.codefactor.io/repository/github/freecodecamp/freecodecamp/overview/main) found multiple issues:

#### Unexpected duplicate &quot;border-color&quot; (declaration-block-no-duplicate-properties)
- [x]  [client\src\components\layouts\global.css:634](https://www.codefactor.io/repository/github/freecodecamp/freecodecamp/source/main/client/src/components/layouts/global.css#L634)
Commit 513f27e408b16d877766882b20167a76f9692729

#### Unexpected duplicate &quot;display&quot; (declaration-block-no-duplicate-properties)
- [x]  [client\src\components\Donation\donation.css:426](https://www.codefactor.io/repository/github/freecodecamp/freecodecamp/source/main/client/src/components/Donation/donation.css#L426)
Commit 421dab38e8083a00c70a00e31ddde55e0bd61b22
- [x]  [client\src\templates\Challenges\video\show.css:61](https://www.codefactor.io/repository/github/freecodecamp/freecodecamp/source/main/client/src/templates/Challenges/video/show.css#L61)

#### Unexpected duplicate &quot;height&quot; (declaration-block-no-duplicate-properties)
- [x]  [client\src\components\Donation\donation.css:318](https://www.codefactor.io/repository/github/freecodecamp/freecodecamp/source/main/client/src/components/Donation/donation.css#L318)
Commit 421dab38e8083a00c70a00e31ddde55e0bd61b22
- [x]  [client\src\components\Header\components\universal-nav.css:9](https://www.codefactor.io/repository/github/freecodecamp/freecodecamp/source/main/client/src/components/Header/components/universal-nav.css#L9)
Commit 3dbe40410cd79fe6bb17f7d85b96b40318afcee9

#### Unexpected duplicate &quot;padding&quot; (declaration-block-no-duplicate-properties)
- [x]  [client\src\templates\Introduction\intro.css:361](https://www.codefactor.io/repository/github/freecodecamp/freecodecamp/source/main/client/src/templates/Introduction/intro.css#L361)
Commit a430aa42ef770ec09d72d64cb6346f91d8e88812

#### Unexpected duplicate &quot;align-items&quot; (declaration-block-no-duplicate-properties)
- [x]  [client\src\components\Header\components\universal-nav.css:108](https://www.codefactor.io/repository/github/freecodecamp/freecodecamp/source/main/client/src/components/Header/components/universal-nav.css#L108)
Commit 3dbe40410cd79fe6bb17f7d85b96b40318afcee9

#### Unexpected duplicate &quot;z-index&quot; (declaration-block-no-duplicate-properties)
- [x]  [client\src\components\Header\components\universal-nav.css:12](https://www.codefactor.io/repository/github/freecodecamp/freecodecamp/source/main/client/src/components/Header/components/universal-nav.css#L12)
Commit 3dbe40410cd79fe6bb17f7d85b96b40318afcee9

#### Unexpected duplicate &quot;position&quot; (declaration-block-no-duplicate-properties)
- [x]  [client\src\templates\Challenges\video\show.css:94](https://www.codefactor.io/repository/github/freecodecamp/freecodecamp/source/main/client/src/templates/Challenges/video/show.css#L94)

### PR DESCRIPTION
**Pull request by Sboonny**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Related #45885

related to first issue, task:
- [x] [client\src\components\layouts\global.css:634](https://www.codefactor.io/repository/github/freecodecamp/freecodecamp/source/main/client/src/components/layouts/global.css#L634)
Commit https://github.com/freeCodeCamp/freeCodeCamp/commit/513f27e408b16d877766882b20167a76f9692729

### CODE REVIEW

**Review by raisedadead — DISMISSED** [review]
Looks good. 

I went ahead and made a few nit-picky changes to the PR title and description (see edit history) for other maintainers to take this PR seriously.

Thanks a lot for taking the time to create the PR. 
Cheers.

File: client/src/components/layouts/global.css [hunk 1]
```diff
@@ -615,7 +615,7 @@ blockquote .small {
   color: #31708f;
   border-color: #31708f;
 }
-.alert .btn:hover,
+
```
Comment (gikf) [inline client/src/components/layouts/global.css:618]:
```suggestion
.alert .btn:hover,
```
I might be a bit confused, but this one doesn't seem to be duplicated.

File: client/src/components/layouts/global.css [hunk 2]
```diff
@@ -615,7 +615,7 @@ blockquote .small {
...
 .alert .btn:focus,
 .alert a:hover,
 .alert a:hover {
```
Comment (gikf) [inline client/src/components/layouts/global.css:621]:
```suggestion
.alert a:hover {
```
This one is.

**Review by gikf — DISMISSED** [review]

File: client/src/components/layouts/global.css (hunk 1 above)
Comment (Sboonny) [reply client/src/components/layouts/global.css:618]: nice catch, you are totally right

**Review by raisedadead — APPROVED** [review]

**Review by naomi-lgbt — APPROVED** [review]

**Pull request merged**